lab-integration/src/main.ts

```diff
@@ -0,0 +1,59 @@
+import dotenv from 'dotenv';
+dotenv.config();
```

Comment:
Can we do `import 'dotenv/config'`?

Ref: https://www.npmjs.com/package/dotenv#%EF%B8%8F-usage